OptimoveSDK/Sources/Classes/OptimoveConfig.swift

```diff
@@ -178,6 +195,21 @@ open class OptimoveConfigBuilder: NSObject {
             Logger.error(error.localizedDescription)
         }
 
+        return self
+    } 
+
+    @discardableResult func setCredentials(
+        optimoveCredentials: String?,
+        optimobileCredentials: String?,
+        preferenceCenterCredentials: String?
+    ) -> OptimoveConfigBuilder {
+        setCredentials(optimoveCredentials: optimoveCredentials, optimobileCredentials: optimobileCredentials)
```

Comment:
this one checks that optimove or optimobile credentials are not nil. so, you cannot use it as setCredentials(nil, nil, credentials)

we need to make sure all these permutations of nullability possibilities make some sense

Reply:
Wait, but do we not want at least one? Or do we want to be able to call with `nil, nil, prefCenterCredentials`?

Reply:
Okay, that is possible after the latest change